examples/cluster/docker-compose.yaml

```diff
@@ -43,7 +42,6 @@ services:
       - dynamodb
     volumes:
       - ../..:/src
-      - /var/run/docker.sock:/var/run/docker.sock
```

Comment:
why is this removed? iirc it's needed for playground to be able to use host's docker.

Reply:
Apparently this is no longer allowed?  I get this error when trying to start the playground:

```
tarting cluster_zookeeper_1 ... done
Starting cluster_dynamodb_1  ... done
Starting cluster_mesosmaster_1 ... done
Starting cluster_mesosagent_1  ... done
ERROR: Cannot create container for service playground: b"You're not allowed to mount /run/docker.sock in rw mode.\nYou are only allowed to bind mount things from these directories: [/nail/home/drmorr:rw /nail/bulkdata:ro /nail/etc:ro /nail/etc/srv-configs:ro /nail/sys/bin/indocker:ro /nail/live/private:ro /nail/srv:ro /nail/home/dockerbuild:ro /nail/scratch/drmorr:rw /scratch/drmorr:rw /etc/boto_cfg:ro /etc/boto.cfg:ro /etc/mrjob.conf:ro /etc/mrjob.cpu-intensive.conf:ro /etc/mrjob.memory-intensive.conf:ro /etc/mrjob.minimal.conf:ro /etc/ssh/ssh_known_hosts:ro /etc/yelp_clog.json:ro /var/run/synapse/services/:ro /var/run/synapse/sockets/:ro /run/synapse/services/:ro /run/synapse/sockets/:ro /etc/nodebot.conf:ro /etc/localtime:ro /usr/share/zoneinfo/:ro /etc/passwd:ro /etc/group:ro /nail/etc/zookeeper_discovery/:ro /nail/etc/kafka_discovery/:ro /nail/tmp/:rw /nail/opt/:ro /etc/send_nsca.cfg:ro /var/run/fusebox/:ro /run/fusebox/:ro]"
```

Reply:
thats a Yelp thing - you need to ``unset DOCKER_HOST``

Reply:
aha thanks.